src/platforms/dart/migration.mdx

```diff
@@ -16,6 +16,10 @@ Future<void> main() async {
 }
 ```
 
+- 'Scope.user' setter was deprecated in favour of `Scope.setUser` and it will be removed in a future update.
```

Comment:
```suggestion
- `Scope.user` setter was deprecated in favor of `Scope.setUser` and it will be removed in a future update.
```

Reply:
I guess `favor` and `favour` are correct, let's @imatwawana help us out :)

Reply:
Favor - we use American spellings in the docs